dev-libs/appstream/appstream-1.0.4.ebuild

```
@@ -51,6 +51,7 @@ BDEPEND="
 	>=sys-devel/gettext-0.19.8
 	doc? ( app-text/docbook-xml-dtd:4.5 )
 	test? ( dev-qt/qttools:6[linguist] )
+	vala? ( dev-lang/vala )
```

```suggestion
	vala? ( $(vala_depend) )
```

without VALA_MAX_API_VERSION etc it doesn't affect much nor with only one slot of vala in tree. But its proper to use it.

---

Updated for both ebuilds, 1.0.3 and 1.0.4, ty